public/components/event_analytics/explorer/context/types.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+import { CoreStart } from '../../../../../../../src/core/public';
+import { OpenSearchDashboardsReactNotifications } from '../../../../../../../src/plugins/opensearch_dashboards_react/public';
+import PPLService from '../../../../services/requests/ppl';
+import DSLService from '../../../../services/requests/dsl';
+import { QueryManager } from '../../../../../common/query_manager';
+
+export type OpenSearchDashboardsServices = Partial<CoreStart>;
+export type DashboardsObservabilityServices = OpenSearchDashboardsServices & {
```

Comment:
`DashboardsObservabilityServices` should contain `CoreStart`. would any services in core start be optional since it is defined as `Partial<CoreStart>`?